<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1076</width>
    <height>674</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout" stretch="10,2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,9">
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="contextMenuPolicy">
           <enum>Qt::NoContextMenu</enum>
          </property>
          <property name="title">
           <string>GroupBox</string>
          </property>
          <widget class="QComboBox" name="serialBaurdRateComboBox">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>50</y>
             <width>69</width>
             <height>22</height>
            </rect>
           </property>
           <item>
            <property name="text">
             <string>9600</string>
            </property>
           </item>
          </widget>
          <widget class="QPushButton" name="openComPortPushButton">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>80</y>
             <width>81</width>
             <height>23</height>
            </rect>
           </property>
           <property name="text">
            <string>打开通信端口</string>
           </property>
          </widget>
          <widget class="QComboBox" name="serialPortsComboBox">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>20</y>
             <width>69</width>
             <height>22</height>
            </rect>
           </property>
          </widget>
          <widget class="Line" name="line">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>110</y>
             <width>118</width>
             <height>3</height>
            </rect>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
          <widget class="QPushButton" name="startPushButton">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>270</y>
             <width>75</width>
             <height>23</height>
            </rect>
           </property>
           <property name="text">
            <string>Start</string>
           </property>
          </widget>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="chartWidget" native="true"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QFrame" name="frame">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <widget class="QWidget" name="">
         <property name="geometry">
          <rect>
           <x>100</x>
           <y>10</y>
           <width>733</width>
           <height>70</height>
          </rect>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <layout class="QGridLayout" name="gridLayout">
            <item row="0" column="1">
             <widget class="QLabel" name="timeLabel_3">
              <property name="text">
               <string>单位</string>
              </property>
             </widget>
            </item>
            <item row="0" column="2">
             <widget class="QLabel" name="timeLabel_7">
              <property name="text">
               <string>单位/格</string>
              </property>
             </widget>
            </item>
            <item row="0" column="3">
             <widget class="QLabel" name="timeLabel_12">
              <property name="text">
               <string>格数</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="timeLabel">
              <property name="text">
               <string>时间</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QComboBox" name="timeUnitComboBox">
              <item>
               <property name="text">
                <string>秒</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>分</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>时</string>
               </property>
              </item>
             </widget>
            </item>
            <item row="1" column="2">
             <widget class="QComboBox" name="timeUnitPerTickComboBox">
              <property name="currentText">
               <string>5</string>
              </property>
              <item>
               <property name="text">
                <string>1</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>2</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>5</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>10</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>20</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>40</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>50</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>100</string>
               </property>
              </item>
             </widget>
            </item>
            <item row="1" column="3">
             <widget class="QComboBox" name="timeTickComboBox">
              <property name="currentText">
               <string>10</string>
              </property>
              <item>
               <property name="text">
                <string>2</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>4</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>5</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>10</string>
               </property>
              </item>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="timeLabel_2">
              <property name="text">
               <string>电流</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QComboBox" name="currentUnitComboBox">
              <item>
               <property name="text">
                <string>mA</string>
               </property>
              </item>
             </widget>
            </item>
            <item row="2" column="2">
             <widget class="QComboBox" name="currentUnitPerTickComboBox">
              <property name="currentText">
               <string>5</string>
              </property>
              <item>
               <property name="text">
                <string>1</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>2</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>5</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>10</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>20</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>50</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>100</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>200</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>500</string>
               </property>
              </item>
             </widget>
            </item>
            <item row="2" column="3">
             <widget class="QComboBox" name="currentTickComboBox">
              <property name="currentText">
               <string>10</string>
              </property>
              <item>
               <property name="text">
                <string>2</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>4</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>5</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>10</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>20</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>188</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item>
               <widget class="QLabel" name="avgCurrentLabel_2">
                <property name="text">
                 <string>  总平均电流</string>
                </property>
               </widget>
              </item>
              <item alignment="Qt::AlignRight|Qt::AlignVCenter">
               <widget class="QLabel" name="avgCurrentLabel">
                <property name="minimumSize">
                 <size>
                  <width>85</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="text">
                 <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                </property>
                <property name="textFormat">
                 <enum>Qt::AutoText</enum>
                </property>
                <property name="scaledContents">
                 <bool>false</bool>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="avgCurrentLabel_3">
                <property name="text">
                 <string>mA</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <item>
               <widget class="QLabel" name="avgCurrentLabel_4">
                <property name="text">
                 <string>选中平均电流</string>
                </property>
               </widget>
              </item>
              <item alignment="Qt::AlignRight|Qt::AlignVCenter">
               <widget class="QLabel" name="selectAreaAvgCurrentLabel">
                <property name="minimumSize">
                 <size>
                  <width>85</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="text">
                 <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="avgCurrentLabel_6">
                <property name="text">
                 <string>mA</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <widget class="QPushButton" name="savePushButton">
              <property name="text">
               <string>保存数据</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="loadPushButton">
              <property name="text">
               <string>载入数据</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1076</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
